docs/usage.md

```diff
@@ -55,6 +55,76 @@ An [example samplesheet](../assets/samplesheet.csv) has been provided with the p
 
 > **NB:** The `group` and `replicate` columns were replaced with a single `sample` column as of v3.1 of the pipeline. The `sample` column is essentially a concatenation of the `group` and `replicate` columns, however it now also offers more flexibility in instances where replicate information is not required e.g. when sequencing clinical samples. If all values of `sample` have the same number of underscores, fields defined by these underscore-separated names may be used in the PCA plots produced by the pipeline, to regain the ability to represent different groupings.
 
+## Reference files
+
+The only reference files required by the pipeline are a FASTA file with the reference genome sequence and a GTF/GFF file with a gene annotation. All other reference files can be created from those by the pipeline.
+However, selecting the appropriate reference genome and annotation to use analysis can still be difficult.
+Here we provide some advice on what is expected by the pipeline:
+
+:::note
+**GENCODE vs ENSEMBL**
+
+Two of the most common sources of genomic references are GENCODE (for mouse and human) and ENSEMBL (for many organisms).
+There has been an effort to standardise information between the two sources and now the references [should be consistent](https://www.gencodegenes.org/pages/faq.html) regardless of where they are obtained from (for mouse and human).
+
+However, while the information is consistent, there are still some practical differences.
+ENSEMBL prefixes chromosome names with `chr` (e.g. `chr1`, `chr1`, ...) while GENCODE uses simple `1`, `2`, etc.
+There can also be different names used for sequences outside the reference chromosomes.
+GENCODE also attaches version identifiers to gene and transcript names (e.g. `ENSG00000254647.1`).
+For these reasons, resources from the two sources cannot be mixed and it is important to stick to one reference source. Some of the steps in the pipeline expect an ENSEMBL reference by default so it is important to set the `--gencode` option if your reference comes from GENCODE.
```

Comment:
```suggestion
**Consistent reference resource usage**

When supplying reference files as discussed below, it is important to be consistent in the reference resource used (Ensembl, Gencode, UCSC etc), since differences in conventions between these resources can make their files incompatible. For example, UCSC prefixes chromosomes with `chr`, while Ensembl does not, so a GTF file from Ensembl should not be supplied alongside a genome FASTA from UCSC.
```

I don't think we need to belabour this point- just make sure people take all their reference files from the same place.

Reply:
Also make sure to integrate with existing content (see the existing Gencode section)